File: Pemrograman1/Modul5/Python/PRAK504-2310817310009-MuhammadFauzanAhsani.py
def reverse(x:int):
    temp = x
    ret = 0

    while (temp > 0):
        ret = (ret * 10) + (temp % 10)
        temp //= 10
    return ret

A, B = map(int, input().split())
A = reverse(A)
B = reverse(B)
C = A + B
print(reverse(C))

File: Pemrograman1/Modul5/C/PRAK504-2310817310009-MuhammadFauzanAhsani.c
#include <stdio.h>

int reverse(int x){ 
    int temp = x;
    int ret = 0;

    while (temp>0){
        ret = (ret* 10) + (temp % 10);
        temp /= 10;
    }
    return ret;}

int main(){
    int A, B;
    scanf("%d %d", &A, &B);
    A = reverse(A);
    B = reverse(B);
    int C = A + B;
    printf("%d", reverse(C));

    return 0;
}